bulbs/poll/migrations/0004_answer_answer_image.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import djbetty.fields
+
+
+class Migration(migrations.Migration):
```

Comment:
@MichaelButkovic I squashed the migrations in #96 

@MelizzaP We'll need to squash the phase 2 migrations here too once #96 is final.